applications: nrf5340_audio: Update changelog for 2.7.0 (#15905)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

nRF5340 Audio
-------------

* Added:

  * CAP initiator for the Bluetooth LE Audio unicast (CIS) client.
  * Support for any context type, not just media.
  * Rejection of connection if unicast_client/broadcast_source tries to use an unsupported sample rate.

>   * Missing data to advertising packet after directed advertising has timed out.

Author: Yes, I didn't see any `Fixed` in the immediate vicinity, so I thought we didn't use it. Will change all the bugfixes to Fixed

  * Debug prints of discovered endpoints.

>   * Support for multiple unicast_servers in unicast_client, regardless of location.

Reviewer: ```suggestion
  * Support for multiple :ref:`unicast servers <nrf53_audio_unicast_server_app>` in :ref:`unicast client <nrf53_audio_unicast_client_app>`, regardless of location.
```

>   * Common Audio Profile (CAP) to unicast_client, including Coordinated Set Identification Profile (CSIP).

Reviewer 2: Is this a partial repeat of the first point?

Author: Fixed

* Removed:

  * The LE Audio controller for nRF5340 library.
    The only supported controller for LE Audio is :ref:`ug_ble_controller_softdevice`.
    This enables use of standard tools for building, configuring, and DFU.

>   * Source PAC records from unicast_server if only sink is supported.

Reviewer: ```suggestion
  * Source PAC records from :ref:`unicast server<nrf53_audio_unicast_server_app>` if only sink is supported.
```
Sink as in broadcast_sink?

Author: No, sink direction for  unicast_server

* Updated:

  * Low latency configuration to be used as default setting for the nRF5340 Audio application.

>   * ACL interval for service discovery to reduce set up time.

Reviewer: ```suggestion
  * ACL interval for service discovery to reduce setup time.
```

>   * Connection procedure so a central will not try to connect to the same device twice.

Reviewer: ```suggestion
  * Connection procedure so that a central does not try to connect to the same device twice.
```

  * Default settings to be lower latency end-to-end.

>   * Presentation delay calculation to be railed between min and max of the unicast_server.

Reviewer: ```suggestion
  * Presentation delay calculation so that it is railed between min and max of the :ref:`unicast server<nrf53_audio_unicast_server_app>`.
```

>   * API for creating a broadcast_source, to be more flexible.

Reviewer: ```suggestion
  * API for creating a :ref:`broadcast source <nrf53_audio_broadcast_source_app>`, to be more flexible.
```
